packages/forklift-console-plugin/src/modules/Plans/views/list/components/PlanStatusCell.tsx

```diff
@@ -103,45 +105,61 @@ export const PlanStatusCell: React.FC<CellProps> = ({ data }) => {
 
   return (
     <Flex alignItems={{ default: 'alignItemsCenter' }} spaceItems={{ default: 'spaceItemsSm' }}>
-      {isPlanLoading ? (
-        <Spinner size="md" />
-      ) : phase === PlanPhase.NotReady ? (
-        t('Validating...')
-      ) : (
-        <Label isCompact>{phase}</Label>
-      )}
-
-      {progressValue !== 0 && isPlanLoading && (
-        <FlexItem className="pf-v5-u-font-size-sm">{Math.trunc(progressValue)}%</FlexItem>
-      )}
-
-      <Split hasGutter>
-        {vmCount.success > 0 && (
-          <SplitItem>
-            <PlanStatusVmCount
-              count={vmCount.success}
-              status="success"
-              linkPath={vmCountLinkPath}
-            />
-          </SplitItem>
-        )}
-
-        {vmCount.canceled > 0 && (
-          <SplitItem>
-            <PlanStatusVmCount
-              count={vmCount.canceled}
-              status="warning"
-              linkPath={vmCountLinkPath}
-            />
-          </SplitItem>
-        )}
-
-        {vmCount.error > 0 && (
-          <SplitItem>
-            <PlanStatusVmCount count={vmCount.error} status="danger" linkPath={vmCountLinkPath} />
-          </SplitItem>
+      <Flex
+        alignItems={{ default: 'alignItemsCenter' }}
+        spaceItems={{ default: 'spaceItemsSm' }}
+        className="plan-status-cell-label-section"
+      >
+        <FlexItem>
+          {isPlanLoading ? (
+            <Spinner size="md" />
+          ) : phase === PlanPhase.NotReady ? (
+            t('Validating...')
+          ) : (
+            <Label isCompact>{phase}</Label>
+          )}
+        </FlexItem>
+
+        {progressValue !== 0 && isPlanLoading && (
+          <FlexItem className="pf-v5-u-font-size-sm">{Math.trunc(progressValue)}%</FlexItem>
         )}
-      </Split>
+      </Flex>
+
+      <FlexItem>
+        <Split hasGutter>
+          {vmCount.success > 0 && (
+            <SplitItem>
+              <PlanStatusVmCount
+                count={vmCount.success}
+                status="success"
+                linkPath={vmCountLinkPath}
+                tooltipLabel={t('Succeeded')}
+              />
+            </SplitItem>
+          )}
+
+          {vmCount.canceled > 0 && (
+            <SplitItem>
+              <PlanStatusVmCount
+                count={vmCount.canceled}
```

Comment:
Why `status="cancel"` is missing?

Reply:
The logic was the Icon component has a specific set of values it takes, and "cancel"/"canceled" isn't one of them. Nor is there one specifically for grey colored icons in general. I've added a 'canceled' status though now to be more specific.